bundles/core/org.eclipse.smarthome.core.test/src/test/java/org/eclipse/smarthome/core/net/HttpServiceUtilTest.java

```diff
@@ -103,10 +100,9 @@ public void shouldReturnUndefinedForException() throws InvalidSyntaxException {
         when(ref2.getProperty(HTTP_PORT)).thenReturn("8080");
         when(ref2.getProperty(Constants.SERVICE_RANKING)).thenReturn("100");
 
-        return Lists.newArrayList(ref1, ref2).toArray(new ServiceReference[0]);
+        return new ServiceReference[] { ref1, ref2 };
```

Comment:
come on, what was wrong with the old implementation? btw, blame says it was me ;-)

Reply:
It worked 😄